resources/scripts/generate-build-data.sh

```diff
@@ -405,6 +406,44 @@ function jqAppend() {
     jq --arg a "${argument}" "${query}" "${file}" > "$tmp" && mv "$tmp" "${file}"
 }
 
+function prepareBuildAnalysis() {
+    key=$1
+    echo "INFO: prepareBuildAnalysis"
+    if [ -e "${PIPELINE_LOG}" ] ; then
```

Comment:
IIUC, this PIPELINE_LOG variable is retrieved by the master from all nodes, right? That's why we can collect all logs in a single point.

Reply:
Sort of, `PIPELINE_LOG` is nothing about the master but an internal variable in the `.sh` script, this particular implementation relies on the BO Rest API -> https://github.com/jenkinsci/blueocean-plugin/tree/master/blueocean-rest#get-log-for-a-pipeline-run

`PIPELINE_LOG` was fetched previously in https://github.com/elastic/apm-pipeline-library/pull/1348/files#diff-27f285536c55231dd2c04cfbafa1df0a2ccd01a8687258180fc9521469f26dcfR486